src/main/java/hu/dpc/rt/kafkastreamer/exporter/KafkaExporterConfiguration.java

```diff
@@ -104,7 +103,15 @@ public static class IndexConfiguration {
     public boolean workflowInstance = true;
     public boolean workflowInstanceCreation = false;
     public boolean workflowInstanceSubscription = false;
-    public boolean processInstance = true;
+//    public boolean processInstance = true;
+
+    public boolean processInstance(final Record<?> record) {
```

Comment:
it would be better to name a method using a verb

Reply:
Updated the function name.